src/wp-admin/edit-form-blocks.php

```diff
@@ -29,6 +29,14 @@
 $current_screen = get_current_screen();
 $current_screen->is_block_editor( true );
 
+// Default to is-fullscreen-mode to avoid jumps in the UI.
+add_filter(
```

Comment:
Ideally all core's `add_filter` and `add_action` calls should be moved to `default-filters.php` I think (or admin-filters).

Reply:
`default-filters.php` is run on every request (or every admin request if `wp-admin/default-filters.php`). We only want to add the `is-fullscreen-mode` class in `wp-admin/edit.php` and `wp-admin/post-new.php`. We use this pattern in a few other admin pages, too.